rules.yaml

```diff
@@ -97,9 +99,12 @@
 - { name: xpi-adblock_plus,            setname: adblock-plus,             repos: [ freebsd ] }
 - { name: xpi-foxyproxy,               setname: foxyproxy,                repos: [ freebsd ] }
 - { name: zookeeper-c,                 setname: zookeeper,                repos: [ gentoo ]  }
+- { namepat: "android-platform-system-(core|extras)", setname: android-tools, repos: [ debian ] }
```

Comment:
adb is part of [platform/system/core](https://android.googlesource.com/platform/system/core/). Debian changed how they package other stuff from that repo, so android-tools is [deprecated](https://wiki.debian.org/AndroidTools/#Original_android-tools_package).